### PR TITLE
dxScheduler - Basic implementation of renovated Month View render

### PR DESCRIPTION
**Pull request by AryamnovEugeniy**

### CODE REVIEW

File: js/renovation/scheduler/workspaces/base/date_table/cell.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+      // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.restAttributes}
+  >
+    {viewModel.props.children}
+  </td>
+);
+
+@ComponentBindings()
+export class DateTableCellBaseProps {
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/base/date_table/cell.tsx:19]: I suggest rename DateTableCellBaseProps -> CellBaseProps

File: js/renovation/scheduler/workspaces/base/date_table/cell.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  @OneWay() className?: string;
+
+  @Slot() children?: any;
+}
+
+@Component({
+  defaultOptionRules: null,
+  view: viewFunction,
+})
+export class DateTableCellBase extends JSXComponent(DateTableCellBaseProps) {}
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/base/date_table/cell.tsx:33]: I suggest rename DateTableCellBase -> CellBase

File: js/renovation/scheduler/workspaces/month/date_table/cell.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  @OneWay() otherMonth?: boolean = false;
+
+  @OneWay() today?: boolean = false;
+}
+
+@Component({
+  defaultOptionRules: null,
+  view: viewFunction,
+})
+export class MonthDateTableCell extends JSXComponent(MonthDateTableCellProps) {
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/month/date_table/cell.tsx:35]: MonthDateTableCell -> MonthCell ?

File: js/renovation/scheduler/workspaces/month/date_table/cell.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    {...viewModel.restAttributes}
+  >
+    <div>
+      {viewModel.props.startDate!.getDate()}
+    </div>
+  </DateTableCellBase>
+);
+
+@ComponentBindings()
+export class MonthDateTableCellProps {
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/month/date_table/cell.tsx:19]: MonthDateTableCellProps -> MonthCellProps

File: js/renovation/scheduler/workspaces/month/date_table/layout.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            />
+          ))}
+        </Row>
+      ))}
+    </tbody>
+  </table>
+);
+
+@ComponentBindings()
+export class MonthDateTableLayoutProps {
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/month/date_table/layout.tsx:38]: MonthDateTableLayoutProps -> MonthLayoutProps

File: js/renovation/scheduler/workspaces/month/date_table/layout.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  @OneWay() viewCellsData?: ViewCellData[][] = [[]];
+
+  @OneWay() className?: string;
+}
+
+@Component({
+  defaultOptionRules: null,
+  view: viewFunction,
+})
+export class MonthDateTableLayout extends JSXComponent(MonthDateTableLayoutProps) {}
```
Comment (AntonSermyazhko) [inline js/renovation/scheduler/workspaces/month/date_table/layout.tsx:48]: MonthDateTableLayout -> MonthLayout ?

**Review by AntonSermyazhko — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Review by pavelgruba — APPROVED** [review]

**Pull request merged**